tests/jvm/src/test/resources/scalalib.expect

```diff
@@ -126,3 +126,19 @@ scala.Null# => abstract final class Null.{+1 decls}
   extends AnyRef
 scala.Null#`<init>`(). => primary ctor <init>: (): Null
   Null => scala.Null#
+
+scala/Singleton.class
+---------------------
+
+Summary:
+Schema => SemanticDB v3
+Uri => scala/Singleton.class
+Text => empty
+Language => Scala
+Symbols => 2 entries
+
+Symbols:
+scala.Singleton# => final trait Singleton.{+1 decls}
+  extends Any
+scala.Singleton#`<init>`(). => primary ctor <init>: (): Singleton
+  Singleton => scala.Singleton#
```

Comment:
not sure about this, should traits have a constructor? I suspect no, right?

Reply:
The singleton trait should not have a constructor. I think traits have synthetic constructors, but you can't reference them in code so there is no need to include it.

Reply:
I've removed it when it's a trait. It still has a constructor from `Any` but it's not a declared member of that trait.